axlearn/experiments/text/gpt/fuji.py

```diff
@@ -91,16 +91,19 @@ class Version(enum.Enum):
         "test": 1 * (1024**4),  # 1T tokens
         "7B": 1 * (1024**4),  # 1T tokens
         "70B": int(1.4 * (1024**4)),  # 1.4T tokens
+        "405B": int(1.4 * (1024**4)),  # 1.4T tokens
```

Comment:
There is no v1/v2 version for 405B right?

Reply:
correct but existing fuji code requires it to be defined otherwise threw an error. Let me clean it up though.